tfsdk/serve_resource_attribute_plan_modifiers_test.go

```diff
@@ -92,6 +92,10 @@ func (rt testServeResourceTypeAttributePlanModifiers) GetSchema(_ context.Contex
 				Type:          types.StringType,
 				PlanModifiers: []AttributePlanModifier{testAttrDefaultValueModifier{}},
 			},
+			"computed_string_no_modifiers": {
```

Comment:
Additional attribute on this resource to allow us to further verify plan unknown marking, since this resource has attribute plan modifiers and allowed me to catch #183 for the future. It does not have to be part of this pull request though. 😄